code/game/objects/items/tanks/tanks.dm

```diff
@@ -224,6 +224,7 @@
 	var/tank_pressure = air_contents.return_pressure()
 	var/actual_distribute_pressure = clamp(tank_pressure, 0, distribute_pressure)
 
+	//Do some algebra here, it'll all make sense I promise
```

Comment:
useless comment

Reply:
It's not a useless comment, it's pointing people towards a common pattern in atmos code, which is taking something like this, 
```
/// kPa*L/(K*mol)
#define R_IDEAL_GAS_EQUATION	8.31
```
and intuiting why the math being done with it works. It might be worth commenting in more detail at a later date, but there's so much of it in atmos code I'd prefer people become comfortable with it

Reply:
your comment doesn't explain that, it just says you're doing algebra--perhaps you can rewrite the ideal gas law equation in this comment, but adjusted to solve for moles

Reply:
sure, I think that'd be a good way to convey the concept

Reply:
It's been updated, lemme know if I was too handholdy with it yeah?